unittests/allplatformstests.py

```diff
@@ -3186,7 +3186,11 @@ def test_summary(self):
         out_start = out.find(expected_lines[0])
         out_lines = out[out_start:].split('\n')[:len(expected_lines)]
         for e, o in zip(expected_lines, out_lines):
-            if e.startswith('    external dep'):
+            if e.startswith('    existing prog'):
+                existing_prog_pattern = re.compile(r'^    existing prog  : (?P<path>.+)$')
+                self.assertRegex(o, existing_prog_pattern)
+                self.assertEqual(Path(re.match(existing_prog_pattern, o).group('path')), Path(sys.executable))
```

Comment:
I do not particularly like this but I am not sure what is responsible for the different capitalization and cannot really debug it efficiently.